pyalgostrategypool/inverse_ema_scalping_v2.py

```diff
@@ -0,0 +1,89 @@
+import talib
+
+from pyalgotrading.constants import *
+from pyalgotrading.strategy.strategy_base import StrategyBase
+
+
+class StrategyInverseEMAScalpingRegularOrder(StrategyBase):
+    name = 'Inverse EMA Scalping Regular Order Strategy v2'
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+        self.larger_time_period = self.strategy_parameters['LARGER_TIME_PERIOD']
+        self.smaller_time_period = self.strategy_parameters['SMALLER_TIME_PERIOD']
+
+        assert (0 < self.larger_time_period == int(self.larger_time_period)), f"Strategy parameter LARGER_TIME_PERIOD should be a positive integer. Received: {self.larger_time_period}"
+        assert (0 < self.smaller_time_period == int(self.smaller_time_period)), f"Strategy parameter SMALLER_TIME_PERIOD should be a positive integer. Received: {self.smaller_time_period}"
+
+        self.main_order_map = None
+
+    def initialize(self):
+        self.main_order_map = {}
+
+    def get_crossover_value(self, instrument):
+        # Get OHLC historical data for the instrument
+        hist_data = self.get_historical_data(instrument)
+
+        # calculate the ema of both time periods
+        larger_ema = talib.EMA(hist_data['close'], timeperiod=self.larger_time_period)
+        smaller_ema = talib.EMA(hist_data['close'], timeperiod=self.smaller_time_period)
+
+        # get crossover value
+        crossover_value = self.utils.crossover(smaller_ema, larger_ema)
+        return crossover_value
+
+    def strategy_select_instruments_for_entry(self, candle, instruments_bucket):
+
+        selected_instruments = []
+        meta = []
+
+        for instrument in instruments_bucket:
+
+            # get crossover value
+            crossover = self.get_crossover_value(instrument)
+            # define key values for action
+            action_constants = {-1: 'BUY', 1: 'SELL'}
```

Comment:
by changing 
`crossover_value = self.utils.crossover(smaller_ema, larger_ema)`
to
`crossover_value = self.utils.crossover(larger_ema, smaller_ema)`

if we are able to change
`action_constants = {-1: 'BUY', 1: 'SELL'}`
to 
`action_constants = {1: 'BUY', -1: 'SELL'}`

then we should do so

wherever possible, let's keep the narrative as 1: BUY and -1: SELL everywhere for crossovers